src/main/webapp/data/userMapData.json

```diff
@@ -1345,6 +1375,7 @@
       "North Carolina School of Science and Mathematics",
       "North Carolina State University",
       "North Central College",
+      "Northeastern University",
```

Comment:
There's already an entry for this 3 lines below.

Reply:
Removed the duplicate entry. 👍

Reply:
The original entry was at the correct position. `Northeastern` should come after `North`.